User_Story#1/SortingApp/Helper/SortType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Helper
+{
+    /// <summary>
+    /// Stores types of sort.
+    /// </summary>
+    /// <owner>Anton Petrenko</owner>
+    public enum SortType
+    {
+        InsertionSort = 0,
```

Comment:
I completely forgot that we need to add comments for each item in enums too.
Sorry for that :).